cmd/types.go

```diff
@@ -105,6 +105,23 @@ type Metadata = map[string]string
 // UnsupportedType is a type alias for v1 version of the k8s apps API, this is meant for testing
 type UnsupportedType = apiv1.Binding
 
+// Resource types listed here
+var resourceTypes = map[string]bool{"ReplicaSet" : true, "Endpoints" : true, "Ingress" : true, "Service" : true,
```

Comment:
I think it's better if we rename this to `ResourceTypes` as we should be exporting this map.